UI/Steps/Steps.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UI.Pages;
+
+namespace UI.Steps
+{
+    public static class Steps
+    {
+        private static LoginPage loginPage = new LoginPage();
+        private static MainPage mainPage = new MainPage();
+
+        public static void Login(string username, string password, int timeSeconds)
+        {
+            loginPage.EnterUsername(username, timeSeconds);
+            loginPage.EnterPassword(password, 1);
+            loginPage.ClickToLoginButton(1);
+        }
+
+        public static void Logout(int timeSeconds)
+        {
+            mainPage.CallUserDropdownMenu(timeSeconds);
+            mainPage.ClickToLogout(2);
```

Comment:
It is better to avoid this hardcoded timeout. Just paste same timeSeconds variable. Do it in function below and above